src/components/QA/QA.jsx

```diff
@@ -77,23 +95,99 @@ class QA extends Component {
     return questions;
   }
 
```

Comment:
Suggestion: I recently learned that you can actually add a name to your events and then use the ES6 computed property technique to set your state variables to the event.target.value (so long as your event names are the same as your state variable names), all with a single event handler. This saves you from having to write out so many dang event handlers. See the attached screenshot for an example.
<img width="1722" alt="Screen Shot 2022-04-09 at 4 01 04 PM" src="https://user-images.githubusercontent.com/51180529/162594370-0be67e18-267b-4adc-a786-7dc6b004edb6.png">

Reply:
ooooo that is so much cleaner. I'll try to refactor when I finish everything else

Reply:
hahah no need to make more work right now. I just thought I'd let you know if you need to add more event handlers in the future!